build.gradle

```diff
@@ -326,22 +322,19 @@ task bundleSecurityAdminStandaloneTarGz(dependsOn: jar, type: Tar) {
 }
 
 task createPluginDescriptor() {
-    if (opensearch_version.contains("-SNAPSHOT")) {
-        opensearch_version=opensearch_version.substring(0, opensearch_version.length() - 9)
-    }
     List<String> descriptorProperties = [
         "description=Provide access control related features for OpenSearch",
-        "version=${version}",
+        "version=${version.tokenize('-')[0]}",
```

Comment:
what would this change achieve?

Reply:
Previously our version string would only be the numeric version `2.0.0` now it can include `2.0.0-alpha` and this is not accepted in the descriptor file.  `String.tokenize(delim)[0]` is a quick way in groovy to substring before the given token.